_clients/data-prepper/pipelines.md

```diff
@@ -40,6 +40,38 @@ simple-sample-pipeline:
 
 - Sinks define where your data goes. In this case, the sink is stdout.
 
+Starting from Data Prepper 2.0, pipelines can be defined across multiple configuration YAML files, where each file contains the configuration for one or more pipelines. This gives you more freedom to organize complex pipeline configurations. Those configuration YAML files should be placed in the `pipelines` folder under the application's home directory (e.g. `/usr/share/data-prepper`) for Data Prepper to load properly.
+{: .note }
+
+## Conditional Routing
+
+Pipelines also supports "conditional routing", a feature that allows you to route Events to different sinks based on specific conditions. To add conditional routing to a pipepline, specify a list of named routes under the `route` component and add specific routes to sinks under the `routes` property. Any sink with the routes property will only accept Events which match at least one of the routing condition. For example: 
```

Comment:
I don't think we should quote conditional route.

I'd just say:

> Pipelines also support conditional route, a feature ...

Also, "supports" should be "support."